### PR TITLE
fix: add imagePullSecrets to cos-fleetshard operators and sync

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,9 @@ bundle/camel-k: operator-sdk
 		"$(CAMEL_K_BUNDLE_NAME)" \
 		"$(CAMEL_K_BUNDLE_NAME)/$(ADDON_VERSION)" \
 		"stable"
+	
+	yq -i 'del(.spec.install.spec.deployments[].label)' \
```
Comment (rinaldodev) [inline Makefile:32]: I don't see those those label fields in any of the CSVs. What is the intention here?

Comment (lburgazzoli) [reply Makefile:32]: mh, looks like those new labels are generated by newer operator-sdk version as they are added with 1.21.0 but not with 1.17.0

Comment (lburgazzoli) [reply Makefile:32]: I've commented out the processing step and added a TODO for future reference

Comment (rinaldodev) [reply Makefile:32]: still there for camel-k and strimzi tho

**Review by rinaldodev — APPROVED** [review]

**Pull request merged**